acceptance/tests/resource/service/ticket_5024_systemd_enabling_masked_service.rb

```diff
@@ -26,7 +26,7 @@
 agents.each do |agent|
   platform = agent.platform.variant
 
-  if agent['platform'] =~ /ubuntu-24.04-amd64/
+  if agent['platform'] =~ /ubuntu-24.04-/
```

Comment:
Not a blocking comment but you might want to future proof this, something like if Ubuntu version is strictly less than 24.04 then use the old path, otherwise use the new path